css/main.css

```diff
@@ -75,7 +75,7 @@ footer {
   }
 }
 
-.cta-button:hover {
+.cta-button,.block:hover {
```

Comment:
I can't really see the block selector used anywhere and have you checked if removing the :hover from the cta-button causes any unexpected change in the UI?

NIT: Can you please add a space between the two selectors' separator (.cta-button, .block:hover)